codecov.yml

```diff
@@ -0,0 +1,16 @@
+# Customizations to codecov for Trillian repo. This will be merged into
+# the team / default codecov yaml file.
+#
+# Validate changes with:
+# curl --data-binary @codecov.yml https://codecov.io/validate
+
+# Exclude code that's for testing, demos or utilities that aren't really
+# part of production releases.
+ignore:
+  - "**/mock_*.go"
+  - "**/testonly"
+  - "docs/**/*"
+  - "examples/**/*"
+  - "integration/**/*"
+  - "testonly/**/*"
```

Comment:
I think this is redundant (I'd expect "**/testonly" to match this).

Reply:
It will take another hour to find out though.

Reply:
I'll merge this and test that out seperately.

Reply:
My test shows that this doesn't match. That might be why testonly wasn't included before if the google.org config (which I don't think I can see) has this in it.